actors/miner/src/lib.rs

```diff
@@ -1720,9 +1745,20 @@ impl Actor {
                 .map_err(|e| {
                     e.downcast_default(ExitCode::USR_ILLEGAL_STATE, "failed to add pre-commit expiry to queue")
                 })?;
+
+
             // Activate miner cron
             needs_cron = !state.deadline_cron_active;
             state.deadline_cron_active = true;
+
+            for sector_num in precommited_sectors.into_iter() {
+                let sector_id = SectorID{
+                    miner: miner_actor_id,
+                    number: sector_num,
+                };
+                emit::sector_precommited(rt, sector_id)?;
+            }
```

Comment:
Move this up a few lines so it's directly after the state updates.

Reply:
Done.